### DISCUSSION TITLE
How to order the facet?

### DISCUSSION THREAD

**Discussion by Weiming113** [GreenleafLab/ArchR#1904] [Questions / Documentation]
<img width="318" alt="Screen Shot 2023-03-29 at 3 49 35 PM" src="https://user-images.githubusercontent.com/125640888/228685101-9b61c4f6-b78a-4194-be25-1d98ef40185e.png">

Hello, I am struggling with how to order the facet and also change the facet name when plotting. Can someone help me solve this?
Thanks

**Comment by rcorces** [comment]
The names are dictated by `groupBy`. The order is alphabetic. The easiest way is to create a new column in `cellColData` that has the names you want preceded by `A_`, `B_` etc to force the ordering.

**Reply by Weiming113** [reply]
Thanks for your reply. It's very helpful.
